taas/config.d/organizations.yml

```diff
@@ -4,30 +4,49 @@ sources:
         options:
             build_by_default: False
         beta-v1:
-            url: https://docs.google.com/spreadsheets/d/1oa7Wknz64dMQlV2XsW0ssjh6Gw5SnHVfGxDSw_5eKOQ/edit#gid=518938746
+            url: https://docs.google.com/spreadsheets/d/1GIA3FrRM1GZS3FoUDNpw35MhC_WLTdKBiiU2tUlWS84/edit#gid=0
             mapping:
-                id: HR info ID
+                id: ID
+                hrinfo_id:
+                    field: HRinfo ID
+                    optional: True
+                rw_id:
+                    field: RW ID
+                    optional: True
+                ocha_gms_id:
+                    field: OCHA_GMS ID
+                    optional: True
                 label: Preferred Term
+                alt_label.default:
```

Comment:
I would suggest having these as `label.humanitarianresponse`, `label.fts`, etc. We know they're alternative labels, because they're not `label.default`.

Our prior art is [countries.yml](https://github.com/UN-OCHA/taas/blob/master/taas/config.d/countries.yml). We should probably match the names which exist in there; some of which are very long because there were arguments for that in PR #14, although they're no longer visible due to isaacs/github#443. :/

`countries.yml` also takes things one step further by auto-populating blank fields with the preferred term. This means, for example, that ReliefWeb could read our code and use `label.reliefweb` without needing to worry null values.

I hold no strong opinions as to whether we should do that here, as it depends upon our use cases. Auto-filling blank fields may make it easier for adoption by individual sites, and countries was being pushed in that direction. At the same time, allowing null fields provides a little more ease when asking "which sites are using a different term than the preferred one"?